opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -561,6 +561,13 @@ message ExponentialHistogramDataPoint {
 
   // max is the maximum value over (start_time, end_time].
   optional double max = 13;
+
+  // ZeroThreshold may be optionally set to convey the width of the
+  // zero region.
+  // When ZeroThreshold is unset or 0, zero count bucket stores
+  // values that cannot be expressed using the standard exponential formula as
+  // well as values that have been rounded to zero.
```

Comment:
Please add "The zero region is defined as the closed interval [-ZeroThreshold, ZeroThreshold]." That is to say that the boundary is inclusive of the threshold on both sides.